src/main/java/com/teragrep/cfe_39/consumers/kafka/queue/WritableQueue.java

```diff
@@ -65,7 +65,7 @@ public class WritableQueue {
     private static final Logger LOGGER = LoggerFactory.getLogger(WritableQueue.class);
```

Comment:
is WritableQueue a proper name for it?

Reply:
Renamed to UniqueFileCreated in commit d3b1a800b3ae0c2a4f46bc9a499a2d590623497d